packages/type-system/ts/integration/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@blockprotocol/type-system-integration",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Integration tests for the Type System package",
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "@blockprotocol/type-system-node": "0.0.0"
```

Comment:
@kachkaev CI seems to be failing because of these
```
error Couldn't find package "@blockprotocol/type-system-node@0.0.0" required by "@blockprotocol/type-system-integration@0.0.0" on the "npm" registry.
```

Should I swap to the `file:/..` syntax? Or is there something else we can do

Reply:
This has been solved by adding the packages to version control